ocs_ci/deployment/hub_spoke.py

```
@@ -521,13 +521,38 @@ def is_fdf_on_provider():
 _fdf_catalog_image_cache = None
 
 
+def _image_ref_matches_repository(image, repository):
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `_image_ref_matches_repository`

(ANN202)